docs/source/build-run-qualcomm-ai-engine-direct-backend.md

```diff
@@ -356,7 +356,7 @@ Please refer to `$EXECUTORCH_ROOT/examples/qualcomm/scripts/` and `EXECUTORCH_RO
 
 ## What is coming?
 
- - [llama2 and llama3](https://github.com/pytorch/executorch/pull/4030). Note that at the moment of writing, we still suffer from the quantization issue in llama2-7B and llama3-8B cases. Only storiesllama works well.
+ - Improve the performance for llama3-8B-Instruct and support bert-mode.
```

Comment:
bert-mode isn't a common word. Batch prefill is probably a better name

Reply:
Thanks for your advice.